src/screens/benefit/Details.tsx

```
@@ -268,15 +268,12 @@ const BenefitsDetails: React.FC = () => {
-			let formData =
-				baseFormData && baseFormData?.dob
-					? {
-							...baseFormData,
-							age:
-								calculateAge(baseFormData.dob) ||
-								baseFormData.age,
-						}
-					: baseFormData;
+			let formData = baseFormData?.dob
+				? {
+						...baseFormData,
```

ESLint

[error] Expected indentation of 5 tabs but found 6.

(indent)